server/routerlicious/packages/lambdas/package.json

```diff
@@ -55,6 +55,8 @@
     "@fluidframework/server-services-core": "^0.1027.0",
     "@types/semver": "^6.0.1",
     "async": "^3.2.0",
+    "axios": "^0.21.1",
+    "crypto-js": "^4.0.0",
```

Comment:
We already use a lightweight sha-1 library in our repo (https://www.npmjs.com/package/sha.js/v/2.4.9). Is it possible to replace cypto-js with that one so bundle size does not increase?